doc/overview.rst

```diff
@@ -74,53 +74,51 @@ Physics
-      |:math:`\gamma`  | Photoelectric       | Livermore                 | :cpp:class:`celeritas::LivermorePEInteractor`      |       0--100 TeV         |
-      |                +---------------------+---------------------------+----------------------------------------------------+--------------------------+
-      |                | Compton scattering  | Klein--Nishina            | :cpp:class:`celeritas::KleinNishinaInteractor`     |       0--100 TeV         |
-      |                +---------------------+---------------------------+----------------------------------------------------+--------------------------+
-      |                | Pair production     | Bethe--Heitler            | :cpp:class:`celeritas::BetheHeitlerInteractor`     |       0--100 TeV         |
-      |                +---------------------+---------------------------+----------------------------------------------------+--------------------------+
-      |                | Rayleigh scattering | Livermore                 | :cpp:class:`celeritas::RayleighInteractor`         |       0--100 TeV         |
-      +----------------+---------------------+---------------------------+----------------------------------------------------+--------------------------+
-
-  .. [1] Multiple Scattering using the Urban Model is only applied up to 100MeV in Celeritas, with no model used above this energy.  
+      +----------------+---------------------+-----------------------------+----------------------------------------------------+--------------------------+
+      | **Particle**   | **Processes**       |  **Models**                 | **Celeritas Implementation**                       | **Applicability**        |
+      +----------------+---------------------+-----------------------------+----------------------------------------------------+--------------------------+
+      | :math:`e^-`    | Ionisation          |  Møller                     | :cpp:class:`celeritas::MollerBhabhaInteractor`     |       0--100 TeV         |
+      |                +---------------------+-----------------------------+----------------------------------------------------+--------------------------+
+      |                | Bremsstrahlung      |  Seltzer--Berger            | :cpp:class:`celeritas::SeltzerBergerInteractor`    |       0--1 GeV           |
+      |                |                     +-----------------------------+----------------------------------------------------+--------------------------+
+      |                |                     |  Relativistic               | :cpp:class:`celeritas::RelativisticBremInteractor` |   1 GeV -- 100 TeV       |
+      |                +---------------------+-----------------------------+----------------------------------------------------+--------------------------+
+      |                | Coulomb scattering  |  Urban                      | :cpp:class:`celeritas::UrbanMscScatter`            |   10 eV -- 100 TeV       |
```

Comment:
Should we update the Urban lower limits to 100 eV if we're using the model energy grid bounds for the applicability?